packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -239,8 +244,19 @@ provideUseId(() => {
   return `${ATTR_KEY}-${instanceId}`
 })
 
-// Provide global security
-provide(GLOBAL_SECURITY_SYMBOL, () => props.parsedSpec.security)
+// Create the workspace store and provide it
+const workspaceStore = createWorkspaceStore({
+  isReadOnly: true,
+  proxyUrl: props.configuration.proxy,
+  themeId: props.configuration.theme,
+  useLocalStorage: false,
+})
+provide(WORKSPACE_SYMBOL, workspaceStore)
+
+// Same for the active entities store
+const activeEntitiesStore = createActiveEntitiesStore(workspaceStore)
```

Comment:
This is just temporary, I will get rid of it in a follow up.... or maybe tomorrow

Reply:
Should we use debounce here, though? Or is it fast enough to not care? Only relevant for docs (and the sandbox, but that doesn’t matter too much) where people might type to change the API defintion.

Reply:
this doesn't really need to be a store, it only is for perf reasons so I think it should be ok